manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Schinagl
+PackageName: Link Shellextension
+License: MIT License
+Copyright: Copyright © 1999-2020

```

```suggestion
Copyright: Copyright © 1999-2021 Hermann Schinagl
```
I believe this is a typo? This is what I got from your previous PR, please commit if you agree with this change.